RiscVPkg/Library/ResetSystemLib/ResetSystemLib.inf

```diff
@@ -0,0 +1,33 @@
+## @file
+#  Library instance for ResetSystem library class for OVMF
+#
+#  Copyright (c) 2020, Hewlett Packard Enterprise Developmente LP. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = ResetSystemLib
+  FILE_GUID                      = 3eff6057-1116-4dcb-837e-c0ef1a120ab1
```

Comment:
This is the newly generated GUID, right? Just make sure this is not duplicate with others.

Reply:
Yes, of course ;)
I have the feeling you have been burnt by this mistake in the past? :laughing:

Reply:
hah, not really.  just people sometime forget to use a new one. :)